opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/MinionRestService.java

```diff
@@ -75,4 +92,27 @@ protected CriteriaBuilder getCriteriaBuilder() {
 	protected JaxbListWrapper<OnmsMinion> createListWrapper(Collection<OnmsMinion> list) {
 		return new OnmsMinionCollection(list);
 	}
+
+    @DELETE
+    @Path("{id}")
+    @Override
+    public Response delete(@PathParam("id") final String id) {
+
+        final OnmsMinion minion = getDao().get(id);
```

Comment:
looks like getDao() need to be overwritten here the base class method is abstract